packages/@aws-cdk/aws-rds/lib/serverless-cluster.ts

```diff
@@ -299,6 +307,21 @@ abstract class ServerlessClusterBase extends Resource implements IServerlessClus
     });
   }
 
+  /**
+   * Grant the given identity to access to the Data API
+   *
+   * @param grantee The principal to grant access to
+   */
+  public grantDataApi(grantee: iam.IGrantable): iam.Grant {
+    const ret = iam.Grant.addToPrincipal({
+      grantee,
+      actions: ['rds-data:*'],
```

Comment:
I'd prefer we list out each of the five actions explicitly, just in case later 'rds-data:nuke-the-world' is introduced and we silently grant all users that permission.

Reply:
Yes, already fixed that 👍